src/frontend/calendar/cell.js

```diff
@@ -31,6 +33,20 @@ function CalendarCell( { blank = false, year, month, day, events } ) {
 					</h3>
 				);
 			} ) }
+
+			{ dayEvents.length > maxEventsToDisplay && (
+				<a
+					href={ '/day/' + date.toISOString() }
+					onClick={ ( e ) => {
+						e.preventDefault();
+
+						// @todo trigger list view
```

Comment:
What about using [a `Dropdown` component](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/dropdown) (or a [DropdownMenu](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/dropdown-menu)) to just continue to show a list of events?